Mobile/ShopLens/Droid/Activities/ShoppingListActivity.cs

```diff
@@ -106,24 +111,36 @@ public void OnResults(Bundle results)
             var matches = results.GetStringArrayList(SpeechRecognizer.ResultsRecognition);
             if (matches.Count > 0)
             {
-                Regex addProductRegex = new Regex(@"^" + ConfigurationManager.AppSettings["CmdAddCartList"] + @"(?:\s\w+)+");
+                string cmdAddProduct = ConfigurationManager.AppSettings["CmdAddCartList"];
+                Regex addProductRegex = new Regex(@"^" + cmdAddProduct + @"(?:\s\w+)+");
 
                 if (matches[0] == voiceListCmd)
```

Comment:
`matches[0]` is used like a bazillion times in this method, why not just assign it to a variable with a meaningful name to improve readability? It has better performance too